python/samples/04.ai.f.dataSource.azureOpenAI/src/config.py

```diff
@@ -0,0 +1,21 @@
+"""
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+"""
+
+import os
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+class Config:
+    """Bot Configuration"""
+
+    PORT = 3978
+    APP_ID = os.environ["BOT_ID"]
+    APP_PASSWORD = os.environ["BOT_PASSWORD"]
+    OPENAI_KEY = os.environ.get("OPENAI_KEY", "")
```

Comment:
can remove this key

Reply:
removed via 6bdae48b29e0846f28d3d7ae0269ab520a3f8ee1